.pre-commit-config.yaml

```diff
@@ -11,6 +11,7 @@ repos:
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
+        exclude: ^mock_data/
```

Comment:
We shouldn't ignore mock_data folder. There are files that should be checked (e.g. yaml file)